src/taskManager/Task.java

```diff
@@ -0,0 +1,89 @@
+package taskManager;
+
+public class Task implements Taskable {
+    private String title;
+    private String description;
+    private String priority;
+
+    //use Enum for status
+    private String status;
+
+    public Task(final String title, final String description, final String priority, final String status) {
+        this.title = title;
+        this.description = description;
+        this.priority = priority;
+        this.status = status;
+    }
+
+    public void setTitle(final String title) {
+        if (title.equals("")) {
```

Comment:
Is the title with whitespaces only considered to be valid? If not, you can trim it before validating

Reply:
Thanks for suggestion!